controllers/tempest_controller.go

```diff
@@ -268,7 +275,7 @@ func (r *TempestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 	customDataConfigMapName := GetCustomDataConfigMapName(instance, externalWorkflowCounter)
 	EnvVarsConfigMapName := GetEnvVarsConfigMapName(instance, externalWorkflowCounter)
 	jobName := r.GetJobName(instance, externalWorkflowCounter)
-	logsPVCName := r.GetPVCLogsName(instance)
+	logsPVCName := r.GetPVCLogsName(instance, externalWorkflowCounter)
```

Comment:
I did not notice it previously but this logic needs to be probably slightly updated. It will not work in case there is `instance.Spec.Parallel == false` and `len(workflow) > 0`. What would happen if we replaced `externalWorkflowCounter` with `workflowStepNum`?

Reply:
Good question. I have tried both cases just to be sure and you are right, the `instance.Spec.Parallel == false` and `len(workflow) > 0` does not work. When I put `workflowStepNum` there instead, it works as expected, so I will do these changes with my next push.